mne/channels/tests/test_channels.py

```diff
@@ -344,6 +344,17 @@ def test_find_ch_adjacency():
     assert ch_names[0] == 'MEG 001'
 
 
+@testing.requires_testing_data
+def test_neuromag122_adjacency():
+    """Test computing the adjacency matrix of Neuromag122-Data"""
+    nm122_fname = op.join(testing.data_path(), 'misc',
+                          'neuromag122_test_file-raw.fif')
+    raw = read_raw_fif(nm122_fname, preload=True)
+    conn, ch_names = find_ch_adjacency(raw.info, 'grad')
+    assert_equal(conn.getnnz(), 1564)
+    assert_equal(len(ch_names), 122)
```

Comment:
We can use plain `assert` and pytest does nice things with it. We tend to reserve `assert_*` functions for array comparisons nowadays.
```suggestion
    assert conn.getnnz() == 1564
    assert len(ch_names) == 122
    assert conn.shape == (122, 122)
```